Different 'abs' declarations conflict with each other

With clang8 on FreeBSD 12 I get these errors:

```
--- casa/CMakeFiles/casa_casa.dir/Arrays/ArrayOpsDiffShapes.cc.o ---
In file included from /usr/ports/astro/casacore/work/casacore-3.1.2/casa/Arrays/ArrayOpsDiffShapes.cc:27:
In file included from /usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/Arrays/ArrayOpsDiffShapes.h:32:
In file included from /usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/Arrays/ArrayMath.h:33:
In file included from /usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/BasicMath/Functors.h:33:
In file included from /usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/BasicSL/Complex.h:36:
/usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/complex.h:40:14: error: target of using declaration conflicts with declaration already in scope
  using std::abs;
             ^
/usr/include/c++/v1/stdlib.h:113:44: note: target of using declaration
inline _LIBCPP_INLINE_VISIBILITY long long abs(long long __x) _NOEXCEPT {return llabs(__x);}
                                           ^
/usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/BasicMath/Math.h:183:16: note: conflicting declaration
  inline Int64 abs(Int64 Val) {return Val;}                                    
               ^
```

gcc8 also complains:
```
--- casa/CMakeFiles/casa_casa.dir/Arrays/Array2Math.cc.o ---
In file included from /usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/Arrays/ArrayMath.h:32,
                 from /usr/ports/astro/casacore/work/casacore-3.1.2/casa/Arrays/Array2Math.cc:28:
/usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/BasicMath/Math.h:183:29: error: 'casacore::Int64 casacore::abs(casacore::Int64)' conflicts with a previous declaration
   inline Int64 abs(Int64 Val) {return Val;}
                             ^
--- casa/CMakeFiles/casa_casa.dir/Arrays/ArrayOpsDiffShapes.cc.o ---
In file included from /usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/Arrays/ArrayMath.h:32,
                 from /usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/Arrays/ArrayOpsDiffShapes.h:32,
                 from /usr/ports/astro/casacore/work/casacore-3.1.2/casa/Arrays/ArrayOpsDiffShapes.cc:27:
/usr/ports/astro/casacore/work/casacore-3.1.2/casacore/casa/BasicMath/Math.h:183:29: error: 'casacore::Int64 casacore::abs(casacore::Int64)' conflicts with a previous declaration
   inline Int64 abs(Int64 Val) {return Val;}
                             ^
```

--- a/casa/BasicMath/Math.h
+++ b/casa/BasicMath/Math.h
@@ -175,17 +175,6 @@ inline Float max(Float a, Float b) { if (a > b) return a; else return b; }
 inline Float min(Float a, Float b) { if (a > b) return b; else return a; }
 // </group>
 
-// Get the absolute value of uInt. Should already be defined
-// for integers in <src><stdlib.h></src>.  Define it for uInts so that certain
-// compilers can resolve the ambiguity when used in a templated class.
-// <group>
-#if defined(AIPS_BSD)                                                          
-  inline Int64 abs(Int64 Val) {return Val;}                                    
-#else                                                                          
-  inline uInt abs(uInt Val) {return Val;}  
-#endif
-// </group>
-
 // Return the square of a value.
 // <group>
 inline Int    square(Int val)    {return val*val;}

